cev_eris.dme

```diff
@@ -15,56 +15,56 @@
 #include "code\hub.dm"
 #include "code\stylesheet.dm"
 #include "code\world.dm"
-#include "code\__DEFINES\_compile_options.dm"
-#include "code\__DEFINES\_globals.dm"
-#include "code\__DEFINES\_planes+layers.dm"
-#include "code\__DEFINES\admin.dm"
-#include "code\__DEFINES\appearance.dm"
-#include "code\__DEFINES\atmos.dm"
-#include "code\__DEFINES\callbacks.dm"
-#include "code\__DEFINES\chemistry.dm"
-#include "code\__DEFINES\colors.dm"
-#include "code\__DEFINES\components.dm"
-#include "code\__DEFINES\craft.dm"
-#include "code\__DEFINES\damage_organs.dm"
-#include "code\__DEFINES\dna.dm"
-#include "code\__DEFINES\economy.dm"
-#include "code\__DEFINES\error_handler.dm"
-#include "code\__DEFINES\flags.dm"
-#include "code\__DEFINES\gamemode.dm"
-#include "code\__DEFINES\integrated_circuit.dm"
-#include "code\__DEFINES\inventory_sizes.dm"
-#include "code\__DEFINES\is_helpers.dm"
-#include "code\__DEFINES\items_clothing.dm"
-#include "code\__DEFINES\jobs.dm"
-#include "code\__DEFINES\lighting.dm"
-#include "code\__DEFINES\machinery.dm"
-#include "code\__DEFINES\materials.dm"
-#include "code\__DEFINES\maths.dm"
-#include "code\__DEFINES\MC.dm"
-#include "code\__DEFINES\misc.dm"
-#include "code\__DEFINES\mob_stats.dm"
-#include "code\__DEFINES\mobs.dm"
-#include "code\__DEFINES\movement.dm"
-#include "code\__DEFINES\nanoui.dm"
-#include "code\__DEFINES\process_scheduler.dm"
-#include "code\__DEFINES\qdel.dm"
-#include "code\__DEFINES\reagents.dm"
-#include "code\__DEFINES\research.dm"
-#include "code\__DEFINES\shields.dm"
-#include "code\__DEFINES\species_languages.dm"
-#include "code\__DEFINES\storyteller_defines.dm"
-#include "code\__DEFINES\subsystems-priority.dm"
-#include "code\__DEFINES\subsystems.dm"
-#include "code\__DEFINES\targeting.dm"
-#include "code\__DEFINES\tick.dm"
-#include "code\__DEFINES\time.dm"
-#include "code\__DEFINES\tools_and_qualities.dm"
-#include "code\__DEFINES\topic.dm"
-#include "code\__DEFINES\turfs.dm"
-#include "code\__DEFINES\unit_testing.dm"
-#include "code\__DEFINES\weapons.dm"
-#include "code\__DEFINES\ZAS.dm"
+#include "code\__defines\_compile_options.dm"

```

Comment:
Can someone please merge <https://github.com/discordia-space/CEV-Eris/pull/3766> so this is no longer an issue?